README.md

```diff
@@ -23,7 +23,8 @@ The official OCS samples are divided in multiple categories depending on the sce
+| **[Grafana](advanced_samples/Grafana)**                                                                                                                                                | Demonstrates how to build a Grafana plugin that retrieves stream data from Sequential Data Store                                                                                                                                                                                                                                                                                                                                   | [NodeJS](advanced_samples/Grafana)                                                                                                                                                                                                                                                                                   | [![Build Status](https://dev.azure.com/osieng/engineering/_apis/build/status/product-readiness/OCS/Grafana_NodeJS?branchName=master)](https://dev.azure.com/osieng/engineering/_build?definitionId=1680&branchName=master)                                                                                                                                                                                                                                                                                                                                                                                                          
```

Comment:
It looks like you deleted the last '|' symbol at the end of this line, I'm guessing that was not intentional?